crates/uv-distribution/src/metadata/lowering.rs

```diff
@@ -731,6 +752,10 @@ fn path_source(
             })
         }
     } else {
+        // TODO(charlie): If a Git repo contains a source that points to a file, what should we do?
+        if git_member.is_some() {
+            return Err(LoweringError::GitFile(url.to_string()));
+        }
```

Comment:
I'm not sure what to do here. We don't have a way to represent this? It's like if you had a Git repo that included a `.whl` file, and another package referred to it via a `path` dependency. We allow that locally, but we can't "translate" that to Git.